clusterloader2/testing/load/deployment.yaml

```diff
@@ -51,6 +67,9 @@ spec:
         - name: configmap
           configMap:
             name: {{.BaseName}}-{{.Index}}
+            items:
```

Comment:
Let's also remove this "items" completely.
If you don't pass items, we will simply mount everything, so given we now have only all-queries, it's simply noop.

Reply:
good to know, done.